controllers/user.js

```diff
@@ -552,7 +552,16 @@ async function getBreachStats(req, res) {
 
 
 function logout(req, res) {
-  req.session.reset();
+  if (req.session.experimentBranch) {
+    // Persist experimentBranch across session reset
+    let experimentBranch = req.session.experimentBranch;
+    req.session.reset();
+    req.session.experimentBranch = experimentBranch;
+  } else {
+    req.session.reset();
+  }
+
+
   res.redirect("/");
```

Comment:
Hmm ... I was going to say get rid of this `else` block, but I know express is a pain about `return` statements when/before using `res.*` methods.

Reply:
Yup. Leaving as is. ☹️